dask/base.py

```diff
@@ -143,8 +144,8 @@ def compute(self, **kwargs):
         """Compute this dask collection
 
         This turns a lazy Dask collection into its in-memory equivalent.
-        For example a Dask.array turns into a  :func:`numpy.array` and a Dask.dataframe
-        turns into a Pandas dataframe.  The entire dataset must fit into memory
+        For example a Dask.array turns into a :func:`numpy.array` and a Dask.dataframe
+        turns into a :func:`pandas.dataframe`.  The entire dataset must fit into memory
```

Comment:
I'm not sure I understand these changes.  I think that you're trying to do cross sphinx linking.  If so, I think that you actually want pandas.DataFrame (note capitalization) and `numpy.ndarray`.  Regardless, if you want to make this change then I encourage you to try building the docs (see https://docs.dask.org/en/latest/develop.html#contributing-to-documentation) and verify that this is an improvement.

Reply:
Oh I wasn't thinking that. I added `:func:` around `pandas.dataframe` simply to keep consistency with what was already done for `numpy.array` on the previous line.

I'll look into the linked resource and test it out.

Reply:
@mrocklin Following instructions in the link above, I wasn't able to see the change take effect
1. Are the [official docs](https://docs.dask.org/en/latest/) based on contents in the `docs` directory of _this repo_?
1. If so, it looks like this particular docstring isn't referenced anywhere in the docs. I did a code search for `For example a Dask.array turns into` throughout the codebase and the only hit was this docstring; explicitly, it's not referenced in any HTML files

Reply:
1.  Yes
2.  In that case it probably doesn't make sense to use sphinx-style entires like this, and we should go with normal human-readable text like `Pandas dataframe` rather than `:func:pandas.dataframe`

Reply:
I agree. Just pushed another commit to address all your feedback. Please re-review.